src/lv_widgets/lv_chart.c

```diff
@@ -1206,7 +1206,7 @@ static void draw_series_line(lv_obj_t * chart, const lv_area_t * series_area, co
             p_act = (start_point + i) % ext->point_cnt;
 
             y_tmp = (int32_t)((int32_t)ser->points[p_act] - ext->ymin[ser->y_axis]) * h;
-            y_tmp = y_tmp / (ext->ymax[ser->y_axis] - ext->ymin[ser->y_axis]);
+            y_tmp = (ext->ymax[ser->y_axis] - ext->ymin[ser->y_axis]);
```

Comment:
Is the removal of `y_tmp /` intentional?

Reply:
No, unintentional, somehow slipped through the diff. Fixed, hopefully without any other silly issues.